health-check/README.md

```diff
@@ -14,3 +22,4 @@ This package helps you implement a standardized `/health` endpoint for your serv
 
 ```bash
 npm install elevate-services-health-check
+npm install kafka-node
```

Comment:
@MallanagoudaB why you have to install kafka node separately? if the package require it it should be included in package.json of the npm package